tailwind.config.mjs

```diff
@@ -50,11 +50,13 @@ export default {
       },
       // Content in the sidebar should never exceed a max-width of `768px`, and
       // that content should be auto-centered
+      // See https://tailwindcss.com/docs/container
       container: {
         center: true,
         // Horizontal padding is larger for wider screens
         padding: {
-          DEFAULT: '0.5rem',
+          // Precise horizontal padding for annotation-card alignment
+          DEFAULT: '9px',
```

Comment:
Net result of this change is 1px additional x-padding outside of toast messages (which was the only place using the `container` class before these changes). i.e. negligible difference, and precisely aligned with annotation cards now.